main.py

```diff
@@ -52,6 +66,13 @@ def build_app(self, *_) -> MDScreenManager:
 
         return manager_screens
 
+    def switch_theme_style(self):
+        self.theme_cls.primary_palette = (
+            "Indigo" if self.theme_cls.primary_palette == "Indigo" else "Indigo"
```

Comment:
This is just a test comment.

Reply:
Can you fix this changes? can you remove that line

Reply:
ok